marbot-cloudfront.yml

```diff
@@ -0,0 +1,274 @@
+---
+# Copyright 2018-2020 widdix GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'marbot.io: CloudFront monitoring (https://github.com/marbot-io/monitoring-jump-start)'
+Metadata:
+  'AWS::CloudFormation::Interface':
+    ParameterGroups:
+    - Label:
+        default: 'marbot endpoint'
+      Parameters:
+      - EndpointId
+      - Stage
+    - Label:
+        default: 'CloudFront'
+      Parameters:
+      - CloudFrontDistributionId
+    - Label:
+        default: 'Thresholds'
+      Parameters:
+      - 5xxErrorRateThreshold
+      - LambdaExecutionErrorThreshold
+      - LambdaValidationErrorThreshold
+      - CacheHitRateThreshold
+      - OriginLatenyThreshold
+Parameters:
+  EndpointId:
+    Description: 'Your marbot endpoint ID (to get this value: select a channel where marbot belongs to and send a message like this: "@marbot show me my endpoint id").'
+    Type: String
+  Stage:
+    Description: 'marbot stage (never change this!).'
+    Type: String
+    Default: v1
+    AllowedValues: [v1, dev]
+  CloudFrontDistributionId:
+    Description: 'The CloudFront distribution ID that you want to monitor.'
+    Type: String
+  5xxErrorRateThreshold:
+    Description: 'The 5xx error rate threshold (set to -1 to disable).'
+    Type: Number
+    Default: 0.01
+    MinValue: -1
+  LambdaExecutionErrorThreshold:
+    Description: 'The Lambda@Edge execution error count threshold (set to >=0 to enable).'
+    Type: Number
+    Default: -1
+    MinValue: -1
+  LambdaValidationErrorThreshold:
+    Description: 'The Lambda@Edge validation error count threshold (set to >=0 to enable).'
```

Comment:
change `set to >=0 to enable` to `set to -1 to disable`
A value of -2 would enable as well at the moment.